egs2/aphasiabank/asr1/run.sh

```diff
@@ -1,28 +1,34 @@
 #!/usr/bin/env bash
-# Set bash to 'debug' mode, it will exit on :
-# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+
+# Aphasia English E-Branchformer experiment
+
 set -e
 set -u
 set -o pipefail
 
+asr_tag="ebranchformer_wavlm"
+
```

Comment:
```suggestion
```
Please standardlize the recipe for the final version by removing some of the set and use default instead.